packages/wrangler/src/api/dev.ts

```diff
@@ -17,6 +17,8 @@ interface DevOptions {
 	nodeCompat?: boolean;
 	compatibilityDate?: string;
 	experimentalEnableLocalPersistence?: boolean;
```

Comment:
I know this is left here to aid the developer experience for those using the experimental flag, but since it is experimental I would just remove it. The developer will get an error message that this flag is not valid, and will quickly be able to work out from the changelog what the correct flag is now.